cmd/exchange_template/exchange_template.go

```diff
@@ -83,37 +77,66 @@ func main() {
 		log.Fatal("GoCryptoTrader: Exchange templating tool stopped...")
 	}
 
-	newExchangeName = strings.ToLower(newExchangeName)
-	v := newExchangeName[:1]
-	capName := strings.ToUpper(v) + newExchangeName[1:]
-
 	exch := exchange{
-		Name:        newExchangeName,
-		CapitalName: capName,
-		Variable:    v,
-		REST:        restSupport,
-		WS:          websocketSupport,
-		FIX:         fixSupport,
+		Name: newExchangeName,
+		REST: restSupport,
+		WS:   websocketSupport,
+		FIX:  fixSupport,
+	}
+
+	if err = makeExchange(&exch); err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Println("GoCryptoTrader: Exchange templating tool service complete")
+	fmt.Println("When the exchange code implementation has been completed (REST/Websocket/wrappers and tests), please add the exchange to engine/exchange.go")
+	fmt.Println("Add the exchange config settings to config_example.json (it will automatically be added to testdata/configtest.json)")
+	fmt.Println("Increment the available exchanges counter in config/config_test.go")
+	fmt.Println("Add the exchange name to exchanges/support.go")
+	fmt.Println("Ensure go test ./... -race passes")
+	fmt.Println("Open a pull request")
+	fmt.Println("If help is needed, please post a message in Slack.")
+}
+
+func checkExchangeName(exchName string) error {
+	if exchName == "" ||
+		exchName == " " ||
+		strings.Contains(exchName, " ") ||
+		len(exchName) == 2 {
```

Comment:
Perhaps make it `<=` as `.\exchange_template.exe -name="s" -rest=true` results in the same issue. This would also capture the first two checks